lambda_proxy/proxy.py

```diff
@@ -541,14 +541,19 @@ def __call__(self, event, context):
         self.log.debug(json.dumps(event.get("headers", {})))
         self.log.debug(json.dumps(event.get("queryStringParameters", {})))
         self.log.debug(json.dumps(event.get("pathParameters", {})))
+        self.log.debug(event.get("path", ""))
 
         self.event = event
         self.context = context
 
         headers = event.get("headers", {}) or {}
         headers = dict((key.lower(), value) for key, value in headers.items())
 
-        resource_path = event.get("path", None)
+        proxy_path = event.get("pathParameters", {}) or {}
```

Comment:
Is this `or {}` redundant to the second `get()` param?

Reply:
going to push some change so this is un relevant, but before there was a problem because sometime `pathParameters` can be present but set to `null` so it then fail to `.get("proxy")` because `null` is not a dict